src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -962,7 +998,6 @@ void Convolution::initDescriptor(const NodeConfig& config) {
             continue;
         for (int n = 0; n < attrsNum; n++) {
```

Comment:
Please also leave a comment for what this iteration over different attrs is about.